### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by Kingsleykingt** [community/community#43619] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm new here. How to get the telegram scraping and adding members software.
I'm using Termux android version

**Comment by osortega** [comment]
Hello @Kingsleykingt! 👋 
I'm not completely sure about the question, are you trying to install a particular tool inside your codespace?

If that's the case, you should follow the linux installing instructions. I'm not familiar with the tools you mentioned above so unfortunately I can't pin-point on how to properly install those.

**Comment by otter-computer** [comment]
Hi @Kingsleykingt,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. We aren't able to help with non-GitHub-related questions.

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
